pkg/networkservice/connectioncontext/dnscontext/utils.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dnscontext
+
+import (
+	"github.com/kelseyhightower/envconfig"
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+	"github.com/sirupsen/logrus"
+)
+
+// ConfigFromEnv returns default getter for DNS configs. Builds config from env variables.
+func ConfigFromEnv() GetDNSConfigsFunc {
```

Comment:
Lets move the config env bits up to the cmd level.

Reply:
Do you mean that it should be located in repo `networkservicemesh/cmd-...` ?

Reply:
Yes...

Reply:
Thanks, fixed.